rewrite-test/src/main/kotlin/org/openrewrite/RecipeTest.kt

```diff
@@ -171,10 +170,9 @@ interface RecipeTest <T: SourceFile> {
     ) {
         assertThat(recipe).`as`("A recipe must be specified").isNotNull
 
-        val source = sources.first()
         val recipeSchedulerCheckingExpectedCycles = RecipeSchedulerCheckingExpectedCycles(ForkJoinScheduler.common(), 0)
         val results = recipe
-            .run(listOf(source),
+            .run(sources,
```

Comment:
I actually don think we want to have dependsOn running through the recipe, as these are really just third-party stubs, similar to a class being included from the classpath.